src/Components/Alert/Components/Filters/Size.tsx

```diff
@@ -0,0 +1,231 @@
+    key: SearchCriteriaAttributeKeys,
+    value: any
+  ) => {
+    dispatch({
+      type: "SET_CRITERIA_ATTRIBUTE",
+      payload: { key: key, value: value },
+    })
+  }
+
+  const toggleSizeSelection = (selected: boolean, name: string) => {
+    let updatedValues = sizes
+
+    if (selected) {
+      updatedValues = [...updatedValues, name]
+    } else {
+      updatedValues = updatedValues.filter(item => item !== name)
+    }
+
+    updateCriteriaAttribute("width", null)
+    updateCriteriaAttribute("height", null)
+    updateCriteriaAttribute("sizes", updatedValues)
+    setCustomSize({ height: DEFAULT_PARSED_RANGE, width: DEFAULT_PARSED_RANGE })
+  }
+
+  const handleSelectMetric = (nextMetric: Metric) => {
+    dispatch({
+      type: "SET_METRIC",
+      payload: nextMetric,
+    })
+    setPredefinedSizes(getPredefinedSizesByMetric(nextMetric))
+
+    if (isCustomRangeValue(width) || isCustomRangeValue(height)) {
+      const customSizeRanges = getCustomSizeRangeInInches(
+        customSize,
+        nextMetric
+      )
+
+      updateCriteriaAttribute("width", customSizeRanges.width)
+      updateCriteriaAttribute("height", customSizeRanges.height)
+    }
+  }
+
+  const handleInputChange = (dimension: "height" | "width", index: number) => ({
+    currentTarget: { value },
+  }: React.FormEvent<HTMLInputElement>) => {
+    const isOpenEnded = value === "" || value === "0"
+    const isMin = index === 0
+
+    setCustomSize(prevCustomSize => {
+      if (isOpenEnded) {
+        prevCustomSize[dimension][index] = "*"
+      } else {
+        const parsedValue = parseInt(value, 10)
+        if (prevCustomSize[dimension])
+          prevCustomSize[dimension][index] = parsedValue
+        else if (isMin) {
+          prevCustomSize[dimension] = [parsedValue, "*"]
+        } else {
+          prevCustomSize[dimension] = ["*", parsedValue]
+        }
+      }
+
+      return { ...prevCustomSize }
+    })
+  }
+
+  const handleSetSizesClick = () => {
+    const customSizeRanges = getCustomSizeRangeInInches(customSize, metric)
+    const width =
+      customSizeRanges.width === DEFAULT_RANGE ? null : customSizeRanges.width
+    const height =
+      customSizeRanges.height === DEFAULT_RANGE ? null : customSizeRanges.height
+
+    updateCriteriaAttribute("width", width)
+    updateCriteriaAttribute("height", height)
+
+    if (
+      customSizeRanges.width != DEFAULT_RANGE ||
+      customSizeRanges.height != DEFAULT_RANGE
+    ) {
+      updateCriteriaAttribute("sizes", [])
+    }
+  }
+
+  return (
+    <>
+      <Join separator={<Spacer y={1} />}>
+        <Text variant="sm-display">Size</Text>
+
+        <RadioGroup
+          defaultValue={metric}
+          onSelect={handleSelectMetric}
+          flexDirection="row"
+          my={2}
+        >
+          <Radio value="cm" label="cm" flex={1} />
+          <Radio value="in" label="in" flex={1} />
+        </RadioGroup>
+
+        <Flex flexDirection="column">
```

Comment:
Looking at the design, this should probably be a two-column design. You can probably use [QuickMultipleSelectAlertFilter](https://github.com/artsy/force/blob/db35f877ccb428046dd1da3df6d3759362be814b/src/Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter.tsx#L14) here.

[Figma Design
](https://www.figma.com/file/lVbIxbnH2IUk1K1FoxyFHV/Filters-and-Alerts-redesign?type=design&node-id=144-32657&mode=design&t=wreNE8xNVYVrRNta-0)
<img width="434" alt="Screenshot 2023-11-23 at 11 18 18" src="https://github.com/artsy/force/assets/4691889/b4366bf4-f948-466c-b562-faa0f57c9881">

Reply:
I think this design is for app specifically, right? I've asked Barney about it back than and he said that the filters on web should be the same as on the live site.

I've pinged him in Slack to review your suggestions, thank you!